src/IssueViz.UnitTests/Models/AnalysisIssueVisualizationTests.cs

```diff
@@ -190,6 +190,22 @@ public void SetSpan_HasSubscribers_NotifiesSubscribers()
             testSubject.Span.Should().Be(newSpan);
         }
 
+        [TestMethod]
+        public void SetIsSuppressed_HasSubscribers_VerifyRaised()
+        {
+            var testSubject = CreateTestSubject();
+            
```

Comment:
I would add an assertion that `IsSuppressed` is `false` to start with, otherwise the test doesn't guarantee that the value has changed.